cmd/root.go

```diff
@@ -95,11 +96,18 @@ func New(config Config) (*cobra.Command, error) {
 	}
 
 	c := &cobra.Command{
-		Use:          name,
-		Short:        description,
-		Long:         description,
-		RunE:         r.Run,
-		SilenceUsage: true,
+		Use:           name,
+		Short:         description,
+		Long:          description,
+		RunE:          r.Run,
+		SilenceUsage:  true,
+		SilenceErrors: true,
+		Args: func(cmd *cobra.Command, args []string) error {
+			if len(args) > 0 {
+				return fmt.Errorf("unknown command %q for %s", args[0], cmd.CommandPath())
+			}
+			return nil
+		},
```

Comment:
This makes the output of an unknown command (e. g. `kgs version` have the correct command in the output -> `kubectl gs` instead of `kubectl\u00a0gs`